README.md

```diff
@@ -90,7 +94,7 @@ A high performance component:
       render: function () {
         return (
           <p>{ this.props.todo.text }</p>
-          <button onClick={this.handleRemove}>Delete</buttion>
+          <button onClick={this.handleRemove}>Delete</button>
         );
       }
     });
```

Comment:
Can we add the `nearestOwnerPropTypes` for the `TodoCreateForm` on line 109 back in:

```
    //you can perform propType checking on nearestOwnerProps too
    nearestOwnerPropTypes: {
      add: React.PropTypes.func.isRequired
    }
```